src/pages/my/todo.tsx

```diff
@@ -1,30 +1,108 @@
 import { GetServerSideProps } from 'next';
-import { Heading } from '@adobe/react-spectrum';
 import { FormattedMessage as Msg } from 'react-intl';
+import { useQuery } from 'react-query';
+import { Flex, Heading, Text } from '@adobe/react-spectrum';
 
+import CallAssignmentList from '../../components/CallAssignmentList';
+import EventList from '../../components/EventList';
+import getBookedEvents from '../../fetching/getBookedEvents';
+import getCallAssignments from '../../fetching/getCallAssignments';
+import getTodoEvents from '../../fetching/getTodoEvents';
+import getUserMemberships from '../../fetching/getUserMemberships';
 import MyHomeLayout from '../../components/layout/MyHomeLayout';
 import { PageWithLayout } from '../../types';
 import { scaffold } from '../../utils/next';
+import { useEventResponses } from '../../hooks';
 
 const scaffoldOptions = {
     localeScope: [
         'layout.my',
+        'misc.callAssignmentList',
+        'misc.eventList',
         'misc.publicHeader',
         'pages.myTodo',
     ],
 };
 
-export const getServerSideProps : GetServerSideProps = scaffold(async () => {
-    return {
-        props: {},
-    };
+export const getServerSideProps : GetServerSideProps = scaffold(async (context) => {
+    const { user } = context;
+
+    let todoEventsState;
+    let bookedEventsState;
+    let callAssignmentsState;
+    let membershipsState;
+
+    if (user) {
+        await context.queryClient.prefetchQuery('todoEvents', getTodoEvents(context.apiFetch));
```

Comment:
Please use better naming than `todoEvents`. See previous comment about responses and bookings.